lpa-submissions-e2e-tests/cypress/integration/accuracyAppellantsSubmission.feature

```diff
@@ -0,0 +1,69 @@
+@wip
+Feature: Accuracy of the Apellants Submission
+
+  As a beta LPAI want to respond whether the information from the appellant accurately reflects the original planning application
+  so that the Planning Inspectorate can make a decision.
+
+  Scenario: AC01 Access 'Review the accuracy of the appellant's submission' sub-section
+    Given the user is on the Task List page
+    When the user selects the link 'Review the accuracy of the appellant's submission'
+    Then the user is presented with the page 'Review the accuracy of the appellant's submission'
+    And the Page Title is 'Review accuracy of appellant's submission - Appeal Questionnaire - Appeal a householder planning decision - GOV.UK'
+
+  Scenario: AC02 LPA User does not select Yes or No
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user does not select an option
+    And the user selects Save and Continue
+    Then the user is shown the error message 'Select yes if the information accurately reflects the planning application'
+    And the user remains in the Accuracy of the appellant's submission sub-section
+
+  Scenario: AC03 LPA user selects Yes and proceeds to task list
+    Given the user is in the Review the accuracy of the appellant's submission sub-section
+    When the user selects Yes
+    And the user selects Save and Continue
+    Then the user continues to the Task List
```

Comment:
Can we change it to **Then the user navigates to the task list** as we have used it in another acceptance criteria in the same feature file (line 63).  As both of them perform same action, we don't need to have different step definitions.

Reply:
I have changed to continues as that is used more commonly and makes more sense in my opinion. The user isn't an active participant in this so they aren't navigating